tests/benchmarks/test_benchmark_vpg.py

```diff
@@ -0,0 +1,256 @@
+from garage.tf.envs import TfEnv
+from garage.tf.experiment import LocalTFRunner
+from garage.tf.policies import GaussianMLPPolicy as TF_GMP
+from garage.torch.algos import VPG as PyTorch_VPG
+from garage.torch.policies import GaussianMLPPolicy as PyTorch_GMP
+from tests import helpers as Rh
+from tests.fixtures import snapshot_config
+
+hyper_parameters = {
+    'hidden_sizes': [64, 64],
+    'center_adv': True,
+    'learning_rate': 1e-2,
+    'discount': 0.99,
+    'n_epochs': 250,
+    'max_path_length': 500
+}
+
+
+def run_garage_pytorch(env, seed, log_dir):
+    """Create garage PyTorch VPG model and training."""
+    env = TfEnv(normalize(env))
+
+    deterministic.set_seed(seed)
+
+    runner = LocalRunner(snapshot_config)
+
+    policy = PyTorch_GMP(
+        env.spec,
+        hidden_sizes=hyper_parameters['hidden_sizes'],
+        hidden_nonlinearity=torch.tanh,
+        # init_std=1e-6,
+        output_nonlinearity=None)
+
+    baseline = LinearFeatureBaseline(env_spec=env.spec)
+
+    algo = PyTorch_VPG(env_spec=env.spec,
+                       policy=policy,
+                       optimizer=torch.optim.Adam,
+                       baseline=baseline,
+                       max_path_length=hyper_parameters['max_path_length'],
+                       discount=hyper_parameters['discount'],
+                       center_adv=hyper_parameters['center_adv'],
+                       policy_lr=hyper_parameters['learning_rate'])
+
+    # Set up logger since we are not using run_experiment
+    tabular_log_file = osp.join(log_dir, 'progress.csv')
+    dowel_logger.add_output(dowel.StdOutput())
+    dowel_logger.add_output(dowel.CsvOutput(tabular_log_file))
+    dowel_logger.add_output(dowel.TensorBoardOutput(log_dir))
+
+    runner.setup(algo, env)
+    runner.train(n_epochs=hyper_parameters['n_epochs'], batch_size=2048)
+
+    dowel_logger.remove_all()
+
+    return tabular_log_file
+
+
+def run_garage_tf(env, seed, log_dir):
+    """Create garage TensorFlow VPG model and training."""
+    deterministic.set_seed(seed)
+
+    with LocalTFRunner(snapshot_config) as runner:
+        env = TfEnv(normalize(env))
+
+        policy = TF_GMP(
+            env_spec=env.spec,
+            hidden_sizes=hyper_parameters['hidden_sizes'],
+            hidden_nonlinearity=tf.nn.tanh,
+            output_nonlinearity=None,
+        )
+
+        baseline = LinearFeatureBaseline(env_spec=env.spec)
+
+        algo = TF_VPG(env_spec=env.spec,
+                      policy=policy,
+                      baseline=baseline,
+                      max_path_length=hyper_parameters['max_path_length'],
+                      discount=hyper_parameters['discount'],
+                      center_adv=hyper_parameters['center_adv'],
+                      optimizer_args=dict(tf_optimizer_args=dict(
+                          learning_rate=hyper_parameters['learning_rate']),
+                                          verbose=True))  # noqa E126
```

Comment:
I put `noqa E126` because yapf keep reformatting this code and Flake8 says "E126 continuation line over-indented for hanging indent"

Reply:
ah. please stick with what flake8 prefers.

please format it how you think it looks best (and satisfies flake8) and then use `# yapf: disable` to prevent YAPF from messing with the code.

Reply:
thanks! fixed